format/Message.fbs

```diff
@@ -62,10 +167,41 @@ table RecordBatch {
   buffers: [Buffer];
 }
 
+/// A data header describing the shared memory layout of a "record" or "row"
+/// batches that supports sparsity and compressoin.
+/// Some systems call this a "sparse row batch" internally and others a
+/// "sparserecord batch".
+table SparseRecordBatch {
+  /// number of records / rows. The arrays in the batch should all have this
+  /// length
+  length: long;
+
+  /// Nodes correspond to the pre-ordered flattened logical schema.  Not all
+  /// fields are necessarily present.  Specifically, "empty" fields may be
+  /// elided if they are either:
+  ///    - a top-level field in the schema,
+  ///    - a child field of a struct field
+  ///    - a child of a union-field, that is not referenced by the type buffer
+  ///    - their parent field was elided.
+  nodes: [SparseFieldNode];
+
+  /// Buffers correspond to the pre-ordered flattened buffer FieldNodeMetadata
```

Comment:
You mean "SparseFieldNode" rather than "FieldNodeMetadata"?

Reply:
yes.  updated.